react/features/overlay/middleware.ts

```diff
@@ -92,20 +94,27 @@ StateListenerRegistry.register(
 
         return configError || connectionError || conferenceError;
     },
-    /* listener */ (error: ErrorType, { dispatch, getState }) => {
+    /* listener */ (error: ErrorType, store: IStore) => {
+        const state = store.getState();
+
         if (!error) {
             return;
         }
 
+        // eslint-disable-next-line no-negated-condition
         if (typeof APP !== 'undefined') {
             APP.API.notifyError({
                 ...error,
-                ...getErrorExtraInfo(getState, error)
+                ...getErrorExtraInfo(state, error)
             });
+        } else {
+            // @ts-ignore
+            store.dispatch(openPageReloadDialog());
```

Comment:
this is already done below, you can drop this else branch since we only want to show the page reload dialog in case of an unrecoverable error